packages/x-date-pickers-pro/src/DateRangePicker/shared.ts

```diff
@@ -67,6 +67,7 @@ export function useDateRangePickerDefaultizedProps<
   };
 }
 
+// What about renaming it `rangePickerValueManager` such that it's clear this manager is common to date, time and dateTime?
```

Comment:
Are you sure that it should be the same ?
`timePickerValueManager` does have some differences with `datePickerValueManager` for instance (to avoid loosing the date).

Reply:
Effectively, I missed the `valueReducer`. Would make more sens if all the value manager implemented the same interface.

DateTime pickers could have `valueReducer = (utils, lastValidDateValue, newDate) => newDate`

For Date Picker, I'm wondering, why don't they need a value reducer to ensure they do not modify time part?

Reply:
To be fair the `valueReducer` is quite a dirty fix and maybe it will not be needed for the new pickers.
I feel like the generic approach of the new fields that never looses the rest of the date is better.

Reply:
I propose we keep it as it is for now and keep the comment. We will see at the end of alpha if a specific method has been needed. Fo now `valueReducer` is useless for range fields because they only modify the selected section of the date.

I tested it with a Date DateTime, and Time range picker fields controlled by the same value, and they stay in sync.

Reply:
Yes the  `valueReducer` will probably be needed for the `TimeRangePicker`.